app/src/org/commcare/android/resource/installers/XFormAndroidInstaller.java

```diff
@@ -76,8 +77,8 @@ protected int customInstall(Resource r, Reference local, boolean upgrade) throws
 
         FormDef formDef;
         try {
-            formDef = new XFormParser(new InputStreamReader(local.getStream(), "UTF-8")).parse();
-        } catch (XFormParseException xfpe) {
+            formDef = XFormExtensionUtils.getFormFromInputStream(local.getStream());

```

Comment:
I didn't name it that because we already have an XFormUtil class somewhere else, but I guess that's ok?